### ISSUE TITLE
Feature: Change NoInteractive flag to NonInteractive

### ISSUE THREAD

**Issue by jodybro** [yonahd/kor#333]
**Is your feature request related to a problem? Please describe.**
No. Purely syntactical 

**Describe the solution you'd like**
The flag to switch to non-interactive mode to be `--non-interactive` vs `--nointeractive`

**Additional context**
First off...I love this tool. It's able to fix things that other colleagues/friends can struggle with for hours in a matter of a few seconds. 
I meant to make this request a while ago when I first found this tool but I completely forgot. 

Basically the first time that I used it and wanted to delete resources noninteractively I typed `--non-interactive` off muscle memory vs the `--no-interactive` flag right now. I even checked the help output beforehand to make sure there was an automated process for resource deletion and everything but still used the former flag since most cli tools that can switch action modes standardize on that name.

**Comment by jodybro** [comment]
Btw if yall want I can handle the refactor since it would be super simple

**Comment by yonahd** [comment]
Hi
Happy you are finding kor useful! 🙂

We're open to this change but we would like to support the old version as well in this upcoming release and just print a deprecation notice on it.

If you would like to take this I'll assign it to you

**Comment by jodybro** [comment]
Yeah that sounds great! Do you have a version in mind to fully deprecate the flag?

**Comment by yonahd** [comment]
@jodybro v0.6.0

**Comment by jodybro** [comment]
Gotcha @yonahd!

**Comment by yonahd** [comment]
@jodybro do you still want to handle this one?
